spec/features/admin/configuration/tax_rates_spec.rb

```diff
@@ -1,12 +1,12 @@
-# frozen_string_literal: true
```

Comment:
Why was this removed?

Reply:
I deleted by mistake - I was not sure what it's doing, so I removed locally but ended up not adding it back; I'll return it there on the next commit.